modules/bcftools/convert/meta.yml

```diff
@@ -53,10 +53,22 @@ output:
       type: file
       description: File containing software versions
       pattern: "versions.yml"
+  - vcf_gz:
```

Comment:
Did you mean to edit this file?

Reply:
yes I forgot to add the output to meta.yml in my last PR so I quickly added it here :)